docs/reference/cat/segments.asciidoc

```diff
@@ -44,6 +44,8 @@ generation::    The generation number is incremented with each segment that is w
                 The name of the segment is derived from this generation number.
 
 docs.count::    The number of non-deleted documents that are stored in this segment.
+                Note that these are Lucene documents, so the count will include root and
```

Comment:
Can we change this to say "will include hidden documents (e.g., from nested types)?"